packages/node/src/transports/base.ts

```diff
@@ -47,30 +57,29 @@ export abstract class BaseTransport implements Transport {
   }
 
   /** Returns a build request option object used by request */
-  protected _getRequestOptions(): http.RequestOptions | https.RequestOptions {
+  protected _getRequestOptions(uri: url.URL): http.RequestOptions | https.RequestOptions {
     const headers = {
-      ...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION),
+      // The auth headers are not included because auth is done via query string to match @sentry/browser.
+      //
+      // ...this._api.getRequestHeaders(SDK_NAME, SDK_VERSION)
       ...this.options.headers,
     };
-    const dsn = this._api.getDsn();
+    const { hostname, port, protocol } = uri;
+    // See https://github.com/nodejs/node/blob/38146e717fed2fabe3aacb6540d839475e0ce1c6/lib/internal/url.js#L1268-L1290
+    const path = `${uri.pathname || ''}${uri.search || ''}`;
```

Comment:
We already deconstruct `uri` 2 lines higer, and cen get `pathname` and `search` there directly.